### PR TITLE
Fixed:Add strings to Translation File-Issue#10656

### LINKED ISSUE #10656: Add strings to Translation File - `_advanced_tagging.html.erb` (2)
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [app/views/tag/_advanced_tagging.html.erb](https://github.com/publiclab/plots2/blob/main/app/views/tag/_advanced_tagging.html.erb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -3,31 +3,31 @@
 </button>
 <div class="dropdown-menu dropdown-menu-right float-right" style="margin-top:5px;" id="pt-list">
   <% if current_user %>
-    <p class="dropdown-header"><b>Recently used tags</b></p>
+    <p class="dropdown-header"><b><%= translation('tag.advanced_tagging.recently_used_tags') %></b></p>
     
   <% if @node && @node.type == "page" %>
-    <a class="dropdown-item" href="#" data-value="parent:">Link to a "parent" page (wikis only)</a>
-    <a class="dropdown-item" href="#" data-value="style:">Add style</a>
-    <a class="dropdown-item" href="#" data-value="lang:">Add Translation widget</a>
+    <a class="dropdown-item" href="#" data-value="parent:"><%= translation('tag.advanced_tagging.link_to_parent') %></a>
+    <a class="dropdown-item" href="#" data-value="style:"><%= translation('tag.advanced_tagging.add_style') %></a>
+    <a class="dropdown-item" href="#" data-value="lang:"><%= translation('tag.advanced_tagging.add_translation_widget') %></a>
   <% else %>
-    <a class="dropdown-item" href="#" data-value="seeks:photos">Request response photos</a>
-    <a class="dropdown-item" href="#" data-value="hidden:response">Hide responses from dashboard</a>
-    <a class="dropdown-item" href="#" data-value="with:">Add Coauthor (notes only)</a>
-    <a class="dropdown-item" href="#" data-value="series:">Mark as part of a series</a>
-    <a class="dropdown-item" href="#" data-value="comment-template:">Add template to the comment form</a>
-    <a class="dropdown-item" target="_blank" href="/wiki/comment-templates">Write a template for the comment form</a>
+    <a class="dropdown-item" href="#" data-value="seeks:photos"><%= translation('tag.advanced_tagging.request_photos') %></a>
+    <a class="dropdown-item" href="#" data-value="hidden:response"><%= translation('tag.advanced_tagging.hide_responses') %></a>
+    <a class="dropdown-item" href="#" data-value="with:"><%= translation('tag.advanced_tagging.add_coauthor') %></a>
+    <a class="dropdown-item" href="#" data-value="series:"><%= translation('tag.advanced_tagging.mark_as_series') %></a>
+    <a class="dropdown-item" href="#" data-value="comment-template:"><%= translation('tag.advanced_tagging.add_template') %></a>
+    <a class="dropdown-item" target="_blank" href="/wiki/comment-templates"><%= translation('tag.advanced_tagging.write_template') %></a>
   <% end %>
   
    <div class="dropdown-divider"></div>
-   <a class="dropdown-item" href="javascript:window.location.reload(true)" onClick="addTag('locked');">Lock Note</a> 
+   <a class="dropdown-item" href="javascript:window.location.reload(true)" onClick="addTag('locked');"><%= translation('tag.advanced_tagging.lock_note') %></a> 
   <% end %>
 
-  <a class="dropdown-item" href="https://publiclab.org/power-tags" target="_blank">Learn About Powertags</a>
+  <a class="dropdown-item" href="https://publiclab.org/power-tags" target="_blank"><%= translation('tag.advanced_tagging.learn_about_powertags') %></a>
 </div>
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by SehgalNeeraj**
Added Strings to Translation File

Fixes #10656   

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by SehgalNeeraj** [comment]
@publiclab/reviewers Please review PR #10659 
8 successful, 1 errored check with Code Climate - any help would be useful

**Review by TildaDares — APPROVED** [review]
Thank you @SehgalNeeraj. This is perfect!

**Comment by TildaDares** [comment]
PR #10658 should be merged first

File: app/views/tag/_advanced_tagging.html.erb [hunk 1]
```diff
@@ -3,31 +3,31 @@
...
-    <a class="dropdown-item" href="#" data-value="with:">Add Coauthor (notes only)</a>
-    <a class="dropdown-item" href="#" data-value="series:">Mark as part of a series</a>
-    <a class="dropdown-item" href="#" data-value="comment-template:">Add template to the comment form</a>
-    <a class="dropdown-item" target="_blank" href="/wiki/comment-templates">Write a template for the comment form</a>
++    <a class="dropdown-item" href="#" data-value="seeks:photos"><%= translation('tag.advanced_tagging.request_photos') %></a>
++    <a class="dropdown-item" href="#" data-value="hidden:response"><%= translation('tag.advanced_tagging.hide_responses') %></a>
++    <a class="dropdown-item" href="#" data-value="with:"><%= translation('tag.advanced_tagging.add_coauthor') %></a>
++    <a class="dropdown-item" href="#" data-value="series:"><%= translation('tag.advanced_tagging.mark_as_series') %></a>
++    <a class="dropdown-item" href="#" data-value="comment-template:"><%= translation('tag.advanced_tagging.add_template') %></a>
++    <a class="dropdown-item" target="_blank" href="/wiki/comment-templates"><%= translation('tag.advanced_tagging.write_template') %></a>
```
Comment (TildaDares) [inline app/views/tag/_advanced_tagging.html.erb:22]: Hi @SehgalNeeraj, can you remove the `+` signs in front of each added line? Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/tag/_advanced_tagging.html.erb (hunk 1 above)
Comment (govindgoel) [reply app/views/tag/_advanced_tagging.html.erb:22]: Yeah @SehgalNeeraj remove the + signs and it is ready to merge then

Comment (SehgalNeeraj) [reply app/views/tag/_advanced_tagging.html.erb]: @TildaDares  @govindgoel  I have made changes and committed to the same PR.
Can you review and suggest next steps.

Comment (govindgoel) [reply app/views/tag/_advanced_tagging.html.erb]: Still it's the same

Screenshot from 2022-01-20 17-37-48

Comment (SehgalNeeraj) [reply app/views/tag/_advanced_tagging.html.erb]: > Still it's the same
> 
> Screenshot from 2022-01-20 17-37-48

hi @govindgoel @TildaDares Just want to be sure if you are reviewing the latest [commit](https://github.com/publiclab/plots2/pull/10659/commits/21158b03bafb04b7ada686d8a9361f6adf8e9f0f). I believe its a presentation thing where new added lines are "shown" with single + sign in github.
The code review comments were to for double + (++) sign and I could see the extra + sign in my repo as well. 
Now its removed based on code review comments.

Worst case, I can just abandon this PR and re-raise a fresh one. Thoughts ?

**Review by govindgoel — APPROVED** [review]

File: app/views/tag/_advanced_tagging.html.erb (hunk 1 above)
Comment (govindgoel) [reply app/views/tag/_advanced_tagging.html.erb]: Now it's fine

**Review by TildaDares — APPROVED** [review]
Thanks @SehgalNeeraj!!

**Pull request merged**